tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -91,8 +91,7 @@ fi
 policy_tests_enabled() {
 	# The Guest images for these platforms have been built using AGENT_POLICY=yes -
 	# see kata-deploy-binaries.sh.
-	[ "${KATA_HYPERVISOR}" == "qemu-sev" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] || \
-		[ "${KATA_HYPERVISOR}" == "qemu-tdx" ] || [ "${KATA_HOST_OS}" == "cbl-mariner" ]
+	[ "${AUTO_GENERATE_POLICY}" == "yes" ]
```

Comment:
In this PR this variable will never be set (because the YAML changes won't be reflected). So we need to find a way to keep the CI working for both these cases:

1. With only script changes taken into account (for this PR's checks).
2. With both script changes and YAML changes taken into account (so that other PRs aren't broken right after you merge this one).